tests/post/features/pods_alive.feature

```diff
@@ -0,0 +1,6 @@
+@liveness @post @ci @local
+Feature: Pods should be alive
+    Scenario: get pods
+        Given The bootstrap phase is executed
+        When I run 'sudo crictl ps -q'
```

Comment:
Let's use `kubectl` or the Python API instead?

Reply:
kubectl is not installed by default but it would make sens to use it for testing more than building sth with pyk8s, will change it